superannotate/__main__.py

```diff
@@ -48,36 +60,51 @@ def ask_token():
 
 
 def main():
-    available_commands = "Available commands to superannotate CLI are: init version create-project create-folder upload-images upload-videos upload-preannotations upload-annotations export-project"
+    available_commands = (
```

Comment:
@VaghinakDev jan,

Instead of global variables we can have the following : 
command_dict = {"create-project":  func}